waves/src/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+import('./native/pkg').then(({setup_lib}) => {
+    setup_lib();
+});
+
+ReactDOM.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+  document.getElementById('root')
+);
+
+// If you want to start measuring performance in your app, pass a function
+// to log results (for example: reportWebVitals(console.log))
+// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
+reportWebVitals();
```

Comment:
I would be interested in keeping this! Can we pass `console.log` here so that we can see how the usage of `wasm` affects performance? Might also be interesting in regards to the async/sync import story.

Reply:
This produces quite some noise in the logs though.  I'll leave it for now to keep the console clean.